Userland/Libraries/LibCore/SocketAddress.h

```diff
@@ -89,7 +120,12 @@ class SocketAddress {
 
 private:
     Type m_type { Type::Invalid };
-    IPv4Address m_ipv4_address;
+
+    union {
+        IPv4Address m_ipv4_address;
+        IPv6Address m_ipv6_address;
```

Comment:
These could be `ipv4` and `ipv6`, the `m_` prefix is not needed here.
So only `m_ip_address.ipv4`

Reply:
done